vagrant_catalog_generator/manage_catalog.py

```diff
@@ -46,10 +46,10 @@ def retrieve_checksum(shafile):
         return checksum_file.read().strip()
 
 
-def generate_box_metadata(boxfiles_directory, box, version, provider, base_url):
+def generate_box_metadata(boxfiles_directory, box, version, provider, base_url, force_generate=False):
     boxfile = path.join(boxfiles_directory, box)
     shafile = boxfile + '.sha256'
-    if not path.isfile(shafile) or stat(shafile).st_size == 0:
+    if not path.isfile(shafile) or stat(shafile).st_size == 0 or force_generate:
```

Comment:
No tests for this? https://github.com/ByteInternet/vagrant-catalog-generator/blob/156ff365b32b609aad57c4088923c1f5c2f888c7/vagrant_catalog_generator/tests/unit/manage_catalog/test_generate_box_metadata.py